### PR TITLE
Tar: Add extraction of file without folder structure

### PR DESCRIPTION
**Pull request by Firesphere**
Most commonly used with e.g. Solr, which has these instructions for installation, but might come in handy for other extraction operations as well.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/tar.md [hunk 1]
```diff
@@ -31,3 +31,7 @@
 - Extract files matching a pattern:
 
 `tar xf {{source.tar}} --wildcards {{"*.html"}}`
+
+- Extract a specific file without the folder structure
```
Comment (zdroid) [inline pages/common/tar.md]:
```suggestion
- Extract a file without making the folder structure:
```

File: pages/common/tar.md [hunk 2]
```diff
@@ -31,3 +31,7 @@
...
+
+`tar xf {{source.tar}} {{source.tar}}/path/to/binary.sh --strip-components={{amount-of-folders-deep}}`
```
Comment (zdroid) [inline pages/common/tar.md]: I don't understand why is `{{source.tar}}/path/to/binary.sh` present here. Anyway, if it stays, it should be put inside `{{}}`.

Comment (Firesphere) [reply pages/common/tar.md]: Yes, the whole example should be between the braces.

File: pages/common/tar.md (hunk 1 above)
Comment (sbrl) [reply pages/common/tar.md]: That still doesn't really make sense @zdroid. What about this?
```suggestion
- Extract a specific file without preserving the folder structure:
```

**Comment by Firesphere** [comment]
Apologies for the delay. I've been quite busy. I'll try to look at the feedback this week!

File: pages/common/tar.md (hunk 2 above)
Comment (agnivade) [reply pages/common/tar.md]: I think the `strip-components` could be explained better. I would say something in the lines of - "Extract a file without creating the given level of directories".

And also, I think just mentioning `--strip-components` is enough. There's no need to extract a single file. The most common usage is to just skip a directory.

Lastly, we use snake_case in our tokens. But we can just shorten it to - {{depth}}

**Comment by Firesphere** [comment]
Sorry stale bot, some virus got in the way. I'll try to get to it soon!

File: pages/common/tar.md [hunk 3]
```diff
@@ -31,3 +31,7 @@
 - Extract files matching a pattern:
 
 `tar xf {{source.tar}} --wildcards {{"*.html"}}`
+
+- Extract a file or folder the folder structure in the archive:
```
Comment (agnivade) [inline pages/common/tar.md]: The description is incorrect. There have been many suggested alternatives in the review. Let's go with one of them.

Comment (Firesphere) [reply pages/common/tar.md]: Agreed, I thought I put that in there, but apparently not. Updated now.

**Pull request merged**